.teamcity/_self/projects/WebApp.kt

```diff
@@ -903,3 +904,154 @@ object RunCalypsoPlaywrightE2eDesktopTests : BuildType({
 		}
 	}
 })
+
+object RunCalypsoPlaywrightE2eMobileTests : BuildType({
+	name = "Playwright E2E tests (mobile)"
+	description = "Runs Calypso e2e tests using Playwright"
+	params {
+		param("use_cached_node_modules", "false")
```

Comment:
Time to get rid of this?

Reply:
I had another branch that looked to get rid of this [here](https://teamcity.a8c.com/buildConfiguration/calypso_RunCalypsoPlaywrightE2eDesktopTests?branch=ci%2Fuse-cached-modules-playwright&buildTypeTab=overview&mode=builds) and it results in the build failing.

I recall either @noahtallen or you discussing how to work with this flag, but I cannot seem to find anything in MGS about this flag anywhere.

Reply:
That branch suggests we need more logging, it is not telling us what broke, right?

```
22:04:46   2021-07-21T20:04:46.871Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.872Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.875Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.875Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.876Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.877Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.881Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.883Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.884Z pw:api => browserType.launch started
22:04:46   2021-07-21T20:04:46.897Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.903Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.903Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.903Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.903Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.908Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.909Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.910Z pw:api <= browserType.launch failed
22:04:46   2021-07-21T20:04:46.912Z pw:api <= browserType.launch failed
```

Reply:
You are correct, it isn't telling us anything useful other than browser launch failing.
I would surmise the binaries for Playwright are being installed correctly, since if they were missing the script wouldn't get that far. 

I can try setting env var `DEBUG=pw:browser,pw:api` to show more logs.